### PR TITLE
[Lab2] Avion - Create (CRUD part 2)

### PR DESCRIPTION
**Pull request by stefinitu**
- Added Create functionality in Service
- Added Create endpoint in Controller

### CODE REVIEW

File: src/main/java/ro/unibuc/hello/service/AvionService.java [hunk 1]
```diff
@@ -25,4 +25,9 @@ public InfoAvion getAvionInfoByNumber(String number) throws EntityNotFoundExcept
         }
         return new InfoAvion(counter.incrementAndGet(), String.format(avionTemplate,entity.number, entity.from, entity.to));
     }
+
+    public InfoAvion addAvion(Avion avion)  {
+        Avion entity = avionRepository.save(avion);
```
Comment (Ana-Mares) [inline src/main/java/ro/unibuc/hello/service/AvionService.java:30]: Maybe we should treat a possible duplicate exception :two_women_holding_hands:  :ambulance:

**Comment by Ana-Mares** [comment]
Neața, dear Ștefania,

Please treat a possible duplicate exception in AvionService two_women_holding_hands ambulance

**Review by Ana-Mares — CHANGES_REQUESTED** [review]

File: src/main/java/ro/unibuc/hello/service/AvionService.java (hunk 1 above)
Comment (stefinitu) [reply src/main/java/ro/unibuc/hello/service/AvionService.java:35]: 🗽

**Review by Ana-Mares — APPROVED** [review]
Tested and it works like a <i>charm<i> :sheep:

**Pull request merged**